CONTRIBUTOR_ROLES.md

```diff
@@ -144,6 +144,12 @@ If an emeritus Maintainer or other retired contributor wants to regain an active
 
 Changes to contributor roles must be approved by a vote of the Oversight Committee or a majority of the current project's Maintainers.
 
+## Communication of contributor role changes
+
+Changes to contributor roles must be communicated by a member of the Oversight Committee to the community. This should be done in the
```

Comment:
"This should be done in the via both Slack and Email" -> I think `in the` is redundant

Reply:
Good catch!